molter/help.py

```diff
@@ -0,0 +1,197 @@
+import functools
+import logging
+
+import dis_snek
```

Comment:
`dev`, and soon `molter` as a whole will use full/direct imports a la `dis-snek` (just so it doesn't look weird when it eventually merges). You can change this if you want, or I can change it once this is merged since it isn't *that* big of a deal.

Reply:
Can change it, sure, tho i assumed all the imports would be changed regardless when this was merged